extra/1-radio-stations.js

```diff
@@ -14,7 +14,13 @@
  */
 
 // `getAllFrequencies` goes here
-
+function getAllFrequencies() {
+  let frequencies = [];
+  for (let i = 87; i <= 108; i++) {
+    frequencies.push(i);
+  }
+  return frequencies;
+}
```

Comment:
great to see you tackling the extra exercises - shows you're really invested in the course! :D